jekyll/_cci2/controlling-serial-execution-across-your-organization.adoc

```diff
@@ -0,0 +1,146 @@
+
+If your job is skipped with a "not run" status, check the following:
+
+* Is another job from a newer pipeline (higher pipeline number) currently running or waiting in the same serial group?
+* Verify your pipeline number is not lower than a currently active pipeline using the same serial group.
+* Remember that the order protection mechanism automatically skips jobs from older pipelines to maintain queue integrity.
+
+=== Jobs not running sequentially?
+
+If jobs are running concurrently when you expected them to run in sequence, check the following:
+
+* Confirm all jobs have the same `serial-group` value, including case sensitivity.
+. Check for typos or whitespace differences in your serial group values.
+. Verify that pipeline parameters are resolving to the expected values in your `serial-group` key.
+. Ensure your serial group value meets the character requirements (alphanumeric plus `.`, `-`, `_`, `/`).
+
+=== Jobs canceled after waiting?
+
+If your jobs are being canceled after waiting in the queue, check the following:
+
+. Are any of your jobs exceeding the five-hour maximum wait time?
+. Investigate if upstream dependencies in your workflow are failing or taking too long.
+. Look for deadlock situations where jobs are waiting for each other to complete.
+
+=== Deployment order appears incorrect?
+
+If you are experiencing deployment order problems, check the following:
+
+. Review the configuration for each job in your serial group to check they are assigned the correct `serial-group` key.
+. Check the pipeline numbers of recent deployments to understand execution order.
+. Consider using more specific serial group naming conventions that include branch or environment information.
+
+=== Still experiencing resource conflicts?
+
+If you are still experiencing resource conflicts despite using serial groups, check the following:
+
+. Make your serial group values more specific by adding project or environment identifiers.
+. Verify that all projects accessing the shared resource are properly configured with the same serial group.
+. Check for jobs outside your serial group that might be accessing the same resources.
+
+== Limitations of serial groups
+
+Serial groups provide powerful orchestration capabilities, but they do have limitations. Understanding these constraints will help you plan your CI/CD strategy more effectively.
+
+=== No workflow-level configuration
+
+Serial groups can only be applied at the job level. You cannot configure an entire workflow to run as a serial group. Each job that needs serialization must have the `serial-group` key individually specified. This means you cannot wrap a collection of jobs with a single serial group declaration.
+
+=== No multiple keys in a single workflow
+
+Using identical serial group values multiple times within the same workflow can result in compilation errors. Each serial group key within a workflow should be unique to prevent conflicts in the execution order.
```

Comment:
This won't be a compilation error, but it might have unexpected behaviour due to the way the jobs join the queue, which can cause more `not_run` jobs than expected.